OmniKit/MessageBlocks/ConfigureAlertsCommand.swift

```diff
@@ -82,18 +95,19 @@ public struct ConfigureAlertsCommand : MessageBlock {
                 let minutes = UInt16(duration.minutes)
                 data.appendBigEndian(minutes)
             }
-            data.appendBigEndian(beepType)
-            
+            data.appendBigEndian(UInt16(beepType.rawValue) << 8 + UInt16(beepRepeat))
```

Comment:
If these are two bytes, you could just append each one in turn, rather than encoding them into a UInt16, and then splitting them back up into something to be appended to the data object.

Reply:
Good idea, this is a bit overkill.